src/Compilers/Core/Portable/Syntax/SeparatedSyntaxList.cs

```diff
@@ -573,5 +573,15 @@ IEnumerator IEnumerable.GetEnumerator()
 
             return SpecializedCollections.EmptyEnumerator<TNode>();
         }
+
+        public static implicit operator SeparatedSyntaxList<SyntaxNode>(SeparatedSyntaxList<TNode> nodes)
```

Comment:
Seems reasonable.

---
In reply to: [65642745](https://github.com/dotnet/roslyn/pull/11415#discussion_r65642745) [](ancestors = 65642745)